### PR TITLE
LambdaExpressionExtensions sample notebook

### PR DESCRIPTION
**Pull request by ElisendaGascon**

### CODE REVIEW

File: Corvus.Extensions.Samples/LambdaExpressionExtensionsSample.dib [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!markdown
+
+## Lambda Expression Extensions = Sample Notebook
```
Comment (jamesbroome) [inline Corvus.Extensions.Samples/LambdaExpressionExtensionsSample.dib:3]: Think that = sign is meant to be a -?

**Comment by jamesbroome** [comment]
My overriding comment on this one is I'm not sure under what conditions I'd need to use it. The samples are easy to understand in the abstract, but I can't think of a use case where I'd need to do this. Are we able to flesh this one out a bit more with something real - i.e. do we have a specific usage scenario that would translate into the notebook?

File: Corvus.Extensions.Samples/LambdaExpressionExtensionsSample.dib (hunk 1 above)
Comment (jesshill99) [reply Corvus.Extensions.Samples/LambdaExpressionExtensionsSample.dib]: Fixed ✅

**Comment by jesshill99** [comment]
> My overriding comment on this one is I'm not sure under what conditions I'd need to use it. The samples are easy to understand in the abstract, but I can't think of a use case where I'd need to do this. Are we able to flesh this one out a bit more with something real - i.e. do we have a specific usage scenario that would translate into the notebook?

Updated the notebook to include examples of specific usage scenarios.

**Review by idg10 — APPROVED** [review]

**Pull request merged**